python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -45,8 +45,8 @@ class Value extends TObject {
     }
 
     /** Gets the origin AST node for this value. */
```

Comment:
This should say "control-flow node" instead of "AST node".

Reply:
Done!

Reply:
@BekaValentine could you make a PR for updating the documentation to not say AST as well :)

Reply:
I don't understand.

Reply:
I'm a bit confused as well.

Reply:
Oh, my bad. I saw the commit changing the **code** from ast -> cfg, and didn't see the commit `Python: ObjectAPI.qll: Fixes docstring` ... never mind me :smile: